db/api/v1/controllers/__tests__/assetDefinitions.controller.test.js

```diff
@@ -27,6 +27,62 @@ describe('assetDefinitions.controller.getAssetTypes', () => {
     });
 });
 
+describe('assetDefinitions.controller.getAssetPropsByTypeID', () => {
+    let req;
+    let res;
+    let next;
+    let expectedAssetTypes;
+    let data;
+
+    beforeEach(() => {
+        req = {
+            'body': {
+                
+            }
+        };
+    
+        res = {
+            json: jest.fn()
+        };
+
+        next = jest.fn();
+        expectedAssetTypes = [{}];
+        data = { rows: expectedAssetTypes };
+        assetDefinitionsDb.findAssetPropsByTypeID = jest.fn(async () => data);
+    });
+
+    it('returns all asset types in DB', async () => {
+        await getAssetPropsByTypeID(req, res, next);
+        expect(res.json).toHaveBeenCalledWith(data);
```

Comment:
Should be testing if the controller pulls the correct parameters out of the req and passes it to the db's getAssetPropsByTypeId

Reply:
@APalumbo97 Is this resolved?

Reply:
@exw4141 Not worrying about the tests right now